initial nutanix doc updates (#4323)
aws/eks-anywhere: docs/content/en/docs/getting-started/production-environment/nutanix-getstarted.md

   ```

   After you have created your `eksa-mgmt-cluster.yaml` and set your credential environment variables, you will be ready to create the cluster.

1. Configure Curated Packages

   The Amazon EKS Anywhere Curated Packages are only available to customers with the Amazon EKS Anywhere Enterprise Subscription. To request a free trial, talk to your Amazon representative or connect with one [here](https://aws.amazon.com/contact-us/sales-support-eks/). Cluster creation will succeed if authentication is not set up, but some warnings may be generated.  Detailed package configurations can be found [here]({{< relref "../../tasks/packages" >}}).

   If you are going to use packages, set up authentication. These credentials should have [limited capabilities]({{< relref "../../tasks/packages/#setup-authentication-to-use-curated-packages" >}}):
   ```bash
   export EKSA_AWS_ACCESS_KEY_ID="your*access*id"
   export EKSA_AWS_SECRET_ACCESS_KEY="your*secret*key"  
   ```
     
1. Create cluster

   ```bash
   eksctl anywhere create cluster \
      # --install-packages packages.yaml \ # uncomment to install curated packages at cluster creation
      -f eksa-mgmt-cluster.yaml
   ```

> 1. Once the cluster is created you can use it with the generated `KUBECONFIG` file in your local directory:

Reviewer: nit
```suggestion
1. Once the cluster is created, you can access it with the generated `KUBECONFIG` file in your local directory:
```

   ```bash
   export KUBECONFIG=${PWD}/${CLUSTER_NAME}/${CLUSTER_NAME}-eks-a-cluster.kubeconfig
   ```
1. Check the cluster nodes:

>    To check that the cluster completed, list the machines to see the control plane, and worker nodes:

Reviewer: ```suggestion
   To check that the cluster is ready, list the machines to see the control plane, and worker nodes:
```

   ```bash

>    kubectl get machines -A

Reviewer: ```suggestion
   kubectl get machines -n eksa-system
```

   ```

   Example command output

>    ```
>    NAMESPACE   NAME                PROVIDERID        PHASE    VERSION
>    eksa-system mgmt-b2xyz          nutanix:/xxxxx    Running  v1.21.2-eks-1-21-5
>    eksa-system mgmt-etcd-r9b42     nutanix:/xxxxx    Running  
>    eksa-system mgmt-md-8-6xr-rnr   nutanix:/xxxxx    Running  v1.21.2-eks-1-21-5
>    ...

Reviewer: Let's use updated info
```suggestion
NAME              CLUSTER  NODENAME                                 PROVIDERID       PHASE     AGE   VERSION
mgmt-4gtt2        mgmt     mgmt-control-plane-1670343878900-2m4ln   nutanix://xxxx   Running   11m   v1.24.7-eks-1-24-4
mgmt-d42xn        mgmt     mgmt-control-plane-1670343878900-jbfxt   nutanix://xxxx   Running   11m   v1.24.7-eks-1-24-4
mgmt-md-0-9868m   mgmt     mgmt-md-0-1670343878901-lkmxw            nutanix://xxxx   Running   11m   v1.24.7-eks-1-24-4
mgmt-md-0-njpk2   mgmt     mgmt-md-0-1670343878901-9clbz            nutanix://xxxx   Running   11m   v1.24.7-eks-1-24-4
mgmt-md-0-p4gp2   mgmt     mgmt-md-0-1670343878901-mbktx            nutanix://xxxx   Running   11m   v1.24.7-eks-1-24-4
mgmt-zkwrr        mgmt     mgmt-control-plane-1670343878900-jrdkk   nutanix://xxxx   Running   11m   v1.24.7-eks-1-24-4
```

   ```

>    The etcd machine doesn't show the Kubernetes version because it doesn't run the kubelet service.

Reviewer: Etcd isn't supported yet. Let's remove all external etcd references for now

Author: Good catch. Done.

1. Check the initial cluster's CRD:

   To ensure you are looking at the initial cluster, list the CRD to see that the name of its management cluster is itself:

   ```bash
   kubectl get clusters mgmt -o yaml
   ```

   Example command output
   ```
   ...

>    kubernetesVersion: "1.21"

Reviewer: ```suggestion
   kubernetesVersion: "1.24"
```

1. Set License Environment Variable

   Add a license to any cluster for which you want to receive paid support. If you are creating a licensed cluster, set and export the license variable (see [License cluster]({{< relref "/docs/tasks/cluster/cluster-license" >}}) if you are licensing an existing cluster):

   ```bash
   export EKSA_LICENSE='my-license-here'
   ```

1. Create a workload cluster

   To create a new workload cluster from your management cluster run this command, identifying:

   * The workload cluster YAML file
   * The initial cluster's credentials (this causes the workload cluster to be managed from the management cluster)

   ```bash
   eksctl anywhere create cluster \
       -f eksa-w01-cluster.yaml  \
       # --install-packages packages.yaml \ # uncomment to install curated packages at cluster creation
       --kubeconfig mgmt/mgmt-eks-a-cluster.kubeconfig
   ```

   As noted earlier, adding the `--kubeconfig` option tells `eksctl` to use the management cluster identified by that kubeconfig file to create a different workload cluster.

1. Check the workload cluster:

   You can now use the workload cluster as you would any Kubernetes cluster.

>    Change your credentials to point to the new workload cluster (for example, `mgmt-w01`), then run the test application with:

Reviewer: consistent cluster name from the previous step
```suggestion
   Change your credentials to point to the new workload cluster (for example, `w01`), then run the test application with:
```

   ```bash

>    export CLUSTER_NAME=mgmt-w01

Reviewer: ```suggestion
   export CLUSTER_NAME=w01
```
